MobileFilter/sections/specific_web.txt

```diff
@@ -95,6 +95,7 @@ m.sport-express.ru##.se-brandedbuttons
 m.ebay-kleinanzeigen.de###srp-sticky
 bakuwaro.com,jiwasoku.com##div[style*="padding: 0px; text-align: center; width: 100%; height: 3"]
 seiga.nicovideo.jp##.ad_320_50
+seiga.nicovideo.jp##.ad_320_100
```

Comment:
Please replace these two rules with `seiga.nicovideo.jp##.ad` i.e. remove
```
seiga.nicovideo.jp##.ad_320_50
seiga.nicovideo.jp##.ad_320_100
```
and add `seiga.nicovideo.jp` to `m.arukikata.co.jp,setn.com,hateblo.jp,hatenablog.com,hatenablog.jp,hatenadiary.com,hatenadiary.jp,hatenadiary.org,syosetu.org##.ad`